### PR TITLE
TOMEE-2707-Translate example datasource-ciphered-password

### PR DESCRIPTION
**Pull request by Daniel-Dos**
link to jira : https://issues.apache.org/jira/browse/TOMEE-2707

### CODE REVIEW

File: examples/datasource-ciphered-password/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+Este exemplo mostra como usar uma senha cifrada com um OpenEJB
+fonte de dados.
+
+Também mostra como implementar sua própria criptografia.
+
+== Configuração
+
+A configuração é simplesmente uma configuração de fonte de dados com um
+parâmetro adicional `` PasswordCipher '' para especificar a criptografia para
```
Comment (robinsonvs) [inline examples/datasource-ciphered-password/README_pt.adoc]: Remove quotation marks `` PasswordCipher ''

File: examples/datasource-ciphered-password/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    // the plain text password is "YouLLN3v3rFindM3"
+    properties.setProperty("ProtectedDatasource.Password", "fEroTNXjaL5SOTyRQ92x3DNVS/ksbtgs");
+    properties.setProperty("ProtectedDatasource.PasswordCipher", "Static3DES");
+    properties.setProperty("ProtectedDatasource.JtaManaged", "true");
+....
+
+== Usando sua própria implementação
+
+O exemplo implementa uma criptografia reversa que simplesmente inverte o
+senha para criptografar/descriptografar.
```
Comment (robinsonvs) [inline examples/datasource-ciphered-password/README_pt.adoc]: replace article "o" with "a"

File: examples/datasource-ciphered-password/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+        return new String(encrypt(new String(encryptedPassword)));
+    }
+}
+....
+
+Para ser funcional, ele precisa do arquivo
+`META-INF/org.apache.openejb.resource.jdbc.PasswordCipher/reverse`.
+
+O nome do arquivo (reverso) define o nome da criptografia a ser usado para o
+Parâmetro PasswordCipher.
```
Comment (robinsonvs) [inline examples/datasource-ciphered-password/README_pt.adoc]: replace "usado" with "usada"

**Comment by robinsonvs** [comment]
just these three suggestions...

**Comment by Daniel-Dos** [comment]
Done thanks

File: examples/datasource-ciphered-password/README_pt.adoc [hunk 4]
```diff
@@ -52,7 +52,7 @@ public static class ReverseEncryption implements PasswordCipher {
 Para ser funcional, ele precisa do arquivo
 `META-INF/org.apache.openejb.resource.jdbc.PasswordCipher/reverse`.
 
-O nome do arquivo (reverso) define o nome da criptografia a ser usado para o
```
Comment (robinsonvs) [inline examples/datasource-ciphered-password/README_pt.adoc]: Hi Daniel, good translation, with just one point of attention, this sentence "nome da criptografia a ser usado" to "nome da criptografia a ser usada"

Comment (Daniel-Dos) [reply examples/datasource-ciphered-password/README_pt.adoc]: Done

File: examples/datasource-ciphered-password/README_pt.adoc [hunk 5]
```diff
@@ -52,7 +52,7 @@ public static class ReverseEncryption implements PasswordCipher {
 Para ser funcional, ele precisa do arquivo
 `META-INF/org.apache.openejb.resource.jdbc.PasswordCipher/reverse`.
 
-O nome do arquivo (reversa) define o nome da criptografia a ser usado para o
+O nome do arquivo (reversa) define o nome da criptografia a ser usada para o
```
Comment (robinsonvs) [inline examples/datasource-ciphered-password/README_pt.adoc]: Hi Daniel, I think "reverse" doesn't need to be translated, for better understanding

Comment (uncommon-design) [reply examples/datasource-ciphered-password/README_pt.adoc]: Thanks for staying on top of this robinsonvs. I think that last one is not a blocker and we should merge it. OK?

**Comment by robinsonvs** [comment]
Nice Daniel

File: examples/datasource-ciphered-password/README_pt.adoc (hunk 5 above)
Comment (Daniel-Dos) [reply examples/datasource-ciphered-password/README_pt.adoc]: Ok thanks.

**Comment by Daniel-Dos** [comment]
Thanks

**Pull request merged**